cmd/av/helpers.go

```diff
@@ -18,7 +18,13 @@ var cachedRepo *git.Repo
 
 func getRepo() (*git.Repo, error) {
 	if cachedRepo == nil {
-		cmd := exec.Command("git", "rev-parse", "--path-format=absolute", "--show-toplevel", "--git-common-dir")
+		cmd := exec.Command(
```

Comment:
Looks like this was just code that was checked in without golines. :shrug:

Reply:
@ErichKramer I think this was you -- can you make sure you have pre-commit setup for this repo?